torch/nn/modules/activation.py

```diff
@@ -126,10 +126,13 @@ class Hardtanh(Module):
         >>> print(m(input))
     """
 
-    def __init__(self, min_value=-1, max_value=1, inplace=False):
+    def __init__(self, min_val=-1, max_val=1, inplace=False, min_value=None, max_value=None):
         super(Hardtanh, self).__init__()
-        self.min_val = min_value
-        self.max_val = max_value
+        self.min_val = min_val
+        self.max_val = max_val
+        assert min_value is None, "keyword argument min_value has been renamed to min_val"
```

Comment:
Do we rather want to add a deprecation warning and just do 
```
if min_value is not None:
    self.min_val = min_value
```
and then fully remove `min_value` and `max_value` in v0.2.x? This way it wouldn't break any existing code unexpectedly?